services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from db.models import Order, Ticket
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user_model = get_user_model()
+    user = user_model.objects.get(username=username)
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+        if date:
+            order.created_at = date
+            order.save()
+
+        for ticket_data in tickets:
+            Ticket.objects.create(
+                movie_session_id=ticket_data["movie_session"],
+                order=order,
+                row=ticket_data["row"],
+                seat=ticket_data["seat"]
+            )
+    return order
+
+
+def get_orders(username: str = None) -> list[Order]:
+    if username:
+        return Order.objects.filter(user__username=username)
+    return Order.objects.all()
```

Comment:
The `get_orders` function allows filtering orders by username, which is useful for retrieving orders specific to a user.